SW-data_analysis-20220111.do

```diff
@@ -40,12 +40,12 @@ egen htn = concat(HLTH_HTN_1_2019 HLTH_HTN_1_2020)
 tab dia
 tab htn
 
-replace dia = "1" if dia == ".1" | dia == "1."| dia == "11"| dia == "12"|  dia == "21"| dia == "51"
-replace dia = "2" if dia == ".." | dia == "5."| dia == "55"|dia == "2."| dia == "25"| dia == "22"| dia == "52"
+replace dia = "1" if strmatch(dia,"*1*")

```

Comment:
just a little tips: you can use strmatch(dia,"*1*") to find all observation in dia that contains "1" anywhere.